### PR TITLE
DX-85 - fix twig code example rendering

### PR DESCRIPTION
**Pull request by Christian-Rades**
Surrounded all code blocks containing twig statements `{% ... %}` with a `{% raw %} ... {% endraw %}` tag to keep gitbook from interpreting the content as markup

### CODE REVIEW

File: guides/plugins/apps/content/cms/add-custom-cms-blocks.md [hunk 1]
```diff
@@ -217,9 +217,12 @@ In `Resources/views/storefront/block/` a Twig template matching the pattern `cms
 
 So in this example, it's sufficient to simply extend the existing `image-text` element:
 {% code title="<app root>/Resources/views/storefront/block/cms-block-swag-image-text-reversed-component.html.twig" %}
+{% raw %}
 ```text
 {% sw_extends '@Storefront/storefront/block/cms-block-image-text.html.twig' %}
 ```
+{% endraw %}
+
```
Comment (elkmod) [inline guides/plugins/apps/content/cms/add-custom-cms-blocks.md:225]: Is this new line needed?

Comment (Christian-Rades) [reply guides/plugins/apps/content/cms/add-custom-cms-blocks.md:225]: No, it's It's an artifact of the fix I'll get rid of it :+1:

**Review by elkmod — APPROVED** [review]

File: guides/plugins/themes/add-icons.md [hunk 2]
```diff
@@ -63,6 +65,7 @@ However, these are not all of your possibilities of configuration. As you see, y
 
 A simple but fully functional example could look like below:
 
+{% raw %}
```
Comment (elkmod) [inline guides/plugins/themes/add-icons.md:68]: Duplicate raws here

File: guides/plugins/themes/add-icons.md [hunk 3]
```diff
@@ -103,13 +107,15 @@ You can define the name of the icon pack and the path to those icons under the `
 ```
 
 You can use your custom icons by specifying your icon pack:
+{% raw %}
```
Comment (elkmod) [inline guides/plugins/themes/add-icons.md:110]: And here

**Review by elkmod — CHANGES_REQUESTED** [review]
Some "already fixed" lines now contain redundant raw elements.

**Pull request closed without merging**